server/server.js

```diff
@@ -150,68 +150,85 @@ app.route('/login')
+      
+        res.send(Mustache.render(chatbotDashboardTemplate, viewParams))
+    } catch (err) {
+        helpers.log(err)
+        res.status(500).send()
+    }
+})
+      
+app.get("/dashboard/:installationId?", async (req, res) => {
+    if (!req.session.user || !req.cookies.user_sid) {
+        res.redirect("/login")
+        return
+    } else if (typeof req.params.installationId !== "string") {
         let installations = await db.getInstallations()
-        res.redirect('/dashboard/' + installations[0].id)
+        res.redirect("/dashboard/" + installations[0].id)
         return
     }
-
+      
     try {
-        let recentSessions = await db.getRecentSessionsWithInstallationId(req.params.installationId)
-        let currentInstallation = await db.getInstallationWithInstallationId(req.params.installationId)
+        let recentSessions = await db.getRecentSessionsWithInstallationId(
+            req.params.installationId
+        )
+        let currentInstallation = await db.getInstallationWithInstallationId(
+            req.params.installationId
+        )
         let allInstallations = await db.getInstallations()
-        
+      
         let viewParams = {
             recentSessions: [],
             currentInstallationName: currentInstallation.name,
             installations: allInstallations
-                .filter(installation => installation.isActive)
-                .map(installation => { 
+                .filter((installation) => installation.isActive)
+                .map((installation) => {
                     return { name: installation.name, id: installation.id }
-                })
-        }
-
-        for(const recentSession of recentSessions) {
+                }),
+        };
+      
+        for (const recentSession of recentSessions) {
             let createdAt = moment(recentSession.createdAt, moment.ISO_8601)
             let updatedAt = moment(recentSession.updatedAt, moment.ISO_8601)
             viewParams.recentSessions.push({
                 unit: recentSession.unit,
-                createdAt: createdAt.tz('America/Vancouver').format('DD MMM Y, hh:mm:ss A'),
-                updatedAt: updatedAt.tz('America/Vancouver').format('DD MMM Y, hh:mm:ss A'),
+                createdAt: createdAt
+                    .tz("America/Vancouver")
+                    .format("DD MMM Y, hh:mm:ss A"),
+                updatedAt: updatedAt
+                    .tz("America/Vancouver")
+                    .format("DD MMM Y, hh:mm:ss A"),
                 state: recentSession.state,
                 numPresses: recentSession.numPresses.toString(),
                 incidentType: recentSession.incidentType,
-                notes: recentSession.notes
+                notes: recentSession.notes,
             })
         }
-        
-        for(let i=0; i<viewParams.recentSessions.length; i++) {
```

Comment:
Why was this row striping removed?

Reply:
Funny. I still see the striping on the list of sessions:
![image](https://user-images.githubusercontent.com/1490437/102286196-a4061c80-3eec-11eb-8200-7d311b1d7193.png)
Was this class just not being used before?

Reply:
Yeah, this should have been removed when I did the styling before, the striping is now done through bootstrap and those classes don't exist in the template file anymore